src/backend/src/services/users.services.ts

```diff
@@ -576,13 +578,34 @@ export default class UsersService {
    * @returns the unread notifications of the user
    */
   static async getUserUnreadNotifications(userId: string, organization: Organization) {
+    const unreadNotifications = await prisma.notification.findMany({
+      where: {
+        users: {
+          some: { userId }
+        }
+      },
+      ...getNotificationQueryArgs(organization.organizationId)
+    });
+
+    if (!unreadNotifications) throw new HttpException(404, 'User Unread Notifications Not Found');
```

Comment:
i think this will throw if this is empty, i dont think we want to do that right, we just want to return the empty list?

Reply:
@caiodasilva2005 this as well